modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/EpicsUtil.scala

```diff
@@ -65,9 +65,8 @@ trait EpicsSystem[T] {
 }
 
 object EpicsCommand {
-  def safe[A](a: SeqAction[A]): SeqAction[A] = EitherT(a.run.attempt.map {
-    case -\/(t) => SeqexecException(t).left
-    case \/-(e) => e
+  def safe[A](a: SeqAction[A]): SeqAction[A] = EitherT(a.run.handle {
+    case e: Exception => SeqexecException(e).left
```

Comment:
In the description you talk about Runtime Errors but this will catch all of them, is that correct?